### ISSUE TITLE
Error with parametric models in fitmodel

### ISSUE THREAD

**Issue by laenan8466** [JeschkeLab/DeerLab#151]
When running the example `ex_fitting_mixed_model.py` from the current state, an error is raised:

```console
Traceback (most recent call last):

  File "[...]\ex_fitting_mixed_model.py", line 68, in <module>
    info = gausswlc()

TypeError: mixedFunction() missing 2 required positional arguments: 'ax' and 'params'
```

**Comment by luisfabib** [comment]
This will be fixed as soon as PR #148 is merged.

**Comment by laenan8466** [comment]
Gnarl, ... 
Memo to me: Look at all current PR, before opening an issue.
